woocommerce/qit-documentation
# Complete documentation rewrite

**PS: Greg, please, don't review this on Saturday, nor Sunday.**

Definetly not done by O1 Pro. Trust me bro.

### What Changed
- Complete reorganization of the documentation structure
- New sidebar navigation
- Consolidated duplicate content (CLI/Dashboard stuff was everywhere)
- Added missing sections for advanced usage and test execution
- Made everything more consistent and interconnected

### Testing Instructions
- Run this branch locally
- See the difference in the organization in the sidebar, see if you like it
- Navigate the documentation
- Compare this proposed documentation with the old documentation, preferably PAGE BY PAGE, as there are some pages in the old documentation with graphics and things like that that were not ported over.. Ideally, we should try to make this new documentation a bit more visual in some parts, with some graphs, etc

### Known TODOs
- Port over graphics from old docs
- Add more visuals where it makes sense (especially for complex concepts)
- Double-check all cross-links work
- Make sure we didn't lose any crucial info in the reorganization

O1 is good but I had to stick with it for over 5h until 2am because I didn't want to lose the huge context I had while generating this doc. I generated this by first figuring out a structure that made sense with it, then generating it page by page, then today, in the morning, coming up with all kinds of crazy scripts to fix the internal links. O1 is good, but I just don't want to make it feel like it's magic. There was a looot of work.
